src/test/java/seedu/address/model/person/NameTest.java

```diff
@@ -27,15 +27,22 @@ public void isValidName() {
         // invalid name
         assertFalse(Name.isValidName("")); // empty string
         assertFalse(Name.isValidName(" ")); // spaces only
-        assertFalse(Name.isValidName("^")); // only non-alphanumeric characters
```

Comment:
I think we can add a test case for names solely wihout a-zA-Z, since that was captured in the regex as well.

Reply:
it exists at line 30